CONTRIBUTING.md

```diff
@@ -0,0 +1,118 @@
+# freeCodeCamp forum clone's contributing guide
+
+## Documentation
+
+Please, make sure to read through the following documents:
+
+- our [code of conduct](./CODE_OF_CONDUCT.md)
+- and [README](./README.md)
+
+## Issues
+
+### Creating issues
+
+Feel free to check out [our codebase](https://github.com/jdwilkin4/fcc-forum-clone). If you think, that something could be added, fixed, or improved on, first that there is no open issue for that by looking through the [issues tab](https://github.com/jdwilkin4/fcc-forum-clone/issues).
```

Comment:
Rephrase: If you think something could be added, fixed, or improved, firstly check that there is no open issue already. You can find it in the [Issues](https://github.com/jdwilkin4/fcc-forum-clone/issues) section of this repository.